DS13/code/modules/overmap/transporter.dm

```diff
@@ -145,6 +190,28 @@
 		T.retrieve(thehewmon)
 	playsound(loc, 'DS13/sound/effects/transporter/retrieve.ogg', 100, 4)
 
+/obj/machinery/computer/camera_advanced/transporter_control/proc/transporters_lock()
+	var/list/mobs = list() //mobs near the eyeobj.
+	for(var/mob/living/thehewmon in orange(eyeobj,3))
```

Comment:
What's a `hewmon`?

Reply:
`in orange(eyeobj,3)` - what does that do?

Reply:
Oranges means in a X tile radius around whatever

Reply:
Oh, in a circle range. Confusing.

Reply:
So what's a `hewmon`?